components/src/pages/form/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Header } from '../../components/header';
+import React, { useEffect, useState } from 'react';
+import { Form } from '../../components/form';
+import { CardListForm } from '../../components/cardListForm';
+import ICardForm from '../../components/cardForm/type';
+import { Modal } from '../../components/modal/formMessage';
+import { DelayedTimeToShowCard, modalShowTime } from '../../managers/timers';
+
+export const FormPage: React.FC = () => {
+  const [cards, cardsState] = useState<ICardForm[]>([]);
+  const [modal, modalState] = useState(false);
+
+  useEffect(() => {
+    if (modal) {
+      setTimeout(() => {
+        modalState(false);
+      }, modalShowTime);
+    }
+  }, [modal]);
```

Comment:
In this case, it's better to show the modal on click rather than on trigger useEffect